arch/csr/Smcntrpmf/mcyclecfg.yaml

```diff
@@ -1,77 +1,71 @@
----
 # yaml-language-server: $schema=../../../schemas/csr_schema.json
 $schema: csr_schema.json#
 kind: csr
 name: mcyclecfg
-long_name: Machine Counter Configuration for Cycle
+long_name: Machine Cycle Counter Configuration
 address: 0x321
 priv_mode: M
 length: 64
+base: 64
 definedBy: Smcntrpmf
 description: |
   The `mcyclecfg` CSR is a 64-bit machine-level register that configures privilege
-  mode filtering for the `mcycle` counter. Each inhibit bit (xINH) suppresses
-  counting of cycles in the corresponding privilege mode when set.
+  mode filtering for the cycle counter. Each inhibit bit (xINH) suppresses
+  counting of events in the corresponding privilege mode when set.
 
-  Bits 62-58 correspond to the following privilege modes:
-    - MINH: M-mode
-    - SINH: S/HS-mode
-    - UINH: U-mode
-    - VSINH: VS-mode
-    - VUINH: VU-mode
+  If set, then counting of events in M-mode is inhibited.
+  If set, then counting of events in S/HS-mode is inhibited.
+  If set, then counting of events in U-mode is inhibited.
+  If set, then counting of events in VS-mode is inhibited.
+  If set, then counting of events in VU-mode is inhibited.
```

Comment:
Hew to the text here, because this is pretty confusing otherwise. Consider using a table, as in the spec. UDB tables can be found in other CSR YAML files by looking for "separator". Each line in the table needs the data from both columns in the spec:

| Field | Description |
| --- | --- |
| MINH | If set, then counting of events in M-mode is inhibited |